src/openzaak/components/zaken/constants.py

```diff
@@ -16,6 +16,22 @@ class BetalingsIndicatie(models.TextChoices):
     geheel = "geheel", _("De met de zaak gemoeide kosten zijn geheel betaald.")
 
 
+class BetaalIndicatie(models.TextChoices):
```

Comment:
Let's add these values to `BetalingsIndicatie` and mark the original fields as deprecated